nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/scheduler/SchedulerServiceImpl.java

```diff
@@ -139,11 +149,33 @@ public void deactivate(ComponentContext context) {
 
     @Override
     public void start(ComponentContext context) {
+        startScheduler();
+    }
+
+    protected void startScheduler() {
+        ClusterService clusterService = Framework.getService(ClusterService.class);
+        Duration duration;
         try {
-            setupScheduler();
-        } catch (IOException | SchedulerException e) {
-            throw new NuxeoException(e);
+            String prop = Framework.getProperty(CLUSTER_START_DURATION_PROP);
+            if (isBlank(prop)) {
+                duration = Duration.ZERO;
+            } else {
+                duration = DurationUtils.parse(prop);
+            }
+        } catch (DateTimeParseException e) {
+            duration = Duration.ZERO;
+        }
+        if (duration.isZero() || duration.isNegative()) {
+            duration = Duration.ofMinutes(1);
         }
```

Comment:
Third time we do the same code, WDYT about adding a method to `ClusterService` or a helper to retrieve the duration from a given Framework property name?

On 10.10 and 9.10, it could be on the `ClusterLockHelper`.

Reply:
How about I put it in `DurationUtils` instead?

Reply:
I pushed 2 more commits for this

Reply:
Even better 👍